book.common/src/main/java/com/happystays/book/common/utils/Constants.java

```diff
@@ -0,0 +1,9 @@
+package com.happystays.book.common.utils;
+
+public final class Constants {
+
+    public static final String BOOK_URL = "http://localhost:8087/hotel-book-service/api/book";
```

Comment:
Can we change this url to the application name registered in service registry for hotel-book-service?